mmv1/products/spanner/InstanceConfig.yaml

```diff
@@ -14,20 +14,106 @@
 --- !ruby/object:Api::Resource
 name: 'InstanceConfig'
 base_url: 'projects/{{project}}/instanceConfigs'
+update_verb: :PATCH
 description: |
   A possible configuration for a Cloud Spanner instance. Configurations
   define the geographic placement of nodes and their replication.
-readonly: true
-# Used as a resource ref
-exclude: true
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/spanner/'
+  api: 'https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instanceConfigs'
+async: !ruby/object:Api::OpAsync
+  actions: ['create', 'update']
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+autogen_async: true
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
```

Comment:
These are the default timeouts; you don't need to include this block unless you want something other than defaults.